### PR TITLE
Document HF_TRANSFER

### PR DESCRIPTION
**Pull request by muellerzr**
This PR documents the HF_TRANSFER feature in the models-downloading.md file.

### CODE REVIEW

**Comment by muellerzr** [comment]
cc @Wauplin

File: docs/hub/models-downloading.md [hunk 1]
```diff
@@ -42,3 +42,16 @@ git clone git@hf.co:<MODEL ID> # example: git clone git@hf.co:bigscience/bloom
...
+## Faster downloads
+
+You can also use the `hf_transfer` library which might give you the ability to download data faster from the Hub. This is 
+mainly relevent if you have a machine with high bandwidth.
+
+```bash
+pip install huggingface_hub[hf_transfer]
+HF_HUB_ENABLE_HF_TRANSFER=1 huggingface-cli download ...
+```
+To learn more, check out the documentation [here](https://huggingface.co/docs/huggingface_hub/guides/download#faster-downloads)
```
Comment (Wauplin) [inline docs/hub/models-downloading.md:55]:
```suggestion

<Tip warning={true}>

`hf_transfer` is a power user tool!
It is tested and production-ready,
but it lacks user-friendly features like advanced error handling or proxies.
For more details, please take a look at this [guide](https://huggingface.co/docs/huggingface_hub/hf_transfer).

</Tip>
```

File: docs/hub/models-downloading.md [hunk 2]
```diff
@@ -42,3 +42,16 @@ git clone git@hf.co:<MODEL ID> # example: git clone git@hf.co:bigscience/bloom
 If you have write-access to the particular model repo, you'll also have the ability to commit and push revisions to the model.
 
 Add your SSH public key to [your user settings](https://huggingface.co/settings/keys) to push changes and/or access private repos.
+
+## Faster downloads
+
+You can also use the `hf_transfer` library which might give you the ability to download data faster from the Hub. This is 
+mainly relevent if you have a machine with high bandwidth.
```
Comment (Wauplin) [inline docs/hub/models-downloading.md:49]:
```suggestion
If you are running on a machine with high bandwidth,
you can increase your download speed with [`hf_transfer`](https://github.com/huggingface/hf_transfer),
a Rust-based library developed to speed up file transfers with the Hub.
```

**Review by Wauplin — COMMENTED** [review]
(small suggestions to be consistent with other parts of the docs)

**Review by Wauplin — APPROVED** [review]
Thanks @muellerzr!

**Review by julien-c — COMMENTED** [review]
ok for me

**Pull request merged**